CRM/Event/PseudoConstant.php

```diff
@@ -298,7 +298,7 @@ public static function &pcPage($id = NULL) {
       );
     }
     if ($id) {
-      return CRM_Utils_Array::value($id, self::$pcPage);
+      return self::$pcPage[$id] ?? NULL;
```

Comment:
@twomice this is the only return-by-ref function affected by this PR. I'll do a followup to remove the `&` from the function declaration per your comment.

Reply:
Turns out the function was unused. #16771